test/test_eager_transforms.py

```diff
@@ -2856,11 +2977,6 @@ def f(x: torch.Tensor) -> torch.Tensor:
     globals(),
     only_for=only_for,
 )
-instantiate_device_type_tests(
-    TestFunctionalize,
-    globals(),
-    only_for=only_for,
-)
```

Comment:
You sure we don't need to test CUDA?

Reply:
There really.... shouldn't be anything cuda-specific about anything going on. I took it out because my local `pytest` is broken and I was too lazy to fix it, and kept forgetting the right "_CPU" incantation to add to the end of my tests when running locally lol. But the tests don't take too long to run, so I may as well add it back.